src/Nancy/PipelineItem.cs

```diff
@@ -1,22 +1,64 @@
 ﻿namespace Nancy
 {
+    /// <summary>
+    /// Defines a pipeline item
+    /// </summary>
+    /// <typeparam name="TDelegate">The type of the delegate.</typeparam>
     public class PipelineItem<TDelegate>
     {
+        /// <summary>
+        /// Gets or sets the pipeline item name.
+        /// </summary>
+        /// <value>
+        /// The name.
+        /// </value>
         public string Name { get; protected set; }
 
+        /// <summary>
+        /// Gets or sets the delegate.
+        /// </summary>
+        /// <value>
+        /// The delegate.
+        /// </value>
         public TDelegate Delegate { get; protected set; }
 
+        /// <summary>
+        /// Initializes a new instance of the <see cref="PipelineItem{TDelegate}"/> class, with
+        /// the provided <paramref name="name"/> and <paramref name="delegate"/>.
+        /// </summary>
+        /// <param name="name">The name.</param>
+        /// <param name="delegate">The delegate.</param>
         public PipelineItem(string name, TDelegate @delegate)
         {
             this.Name = name;
             this.Delegate = @delegate;
         }
 
+        /// <summary>
+        /// Performs an implicit conversion from <see>

```

Comment:
Same as previous comments about the use of `<see>` and `<cref>` => `<see cref="..."/>`